Runtime_Support/dot_net_core.md

```diff
@@ -10,6 +10,14 @@ Once a version of .NET Core has reached it's end of life (EOL) it will no longer
 
 Existing applications configured to target a runtime version that has reached EOL should not be affected.
 
+## .NET 5 Early Access limitations
+
+During the early access period, .NET 5 apps will be subject to the **Early Access** limitations listed in the [early access support document](./early_access.md).
+
+[Application Insights](https://azure.microsoft.com/services/monitor) is currently not supported for .NET 5 apps, we are working with the Application Insights team to resolve this.
```

Comment:
cc @shirhatti as an FYI that @btardif's article on early access is going to call out the lack of support for App Insights. I wanted you to review this. I'll be linking to Byron's doc here and the one on general early access info, so I don't know how deep i'll go in the actual .NET blog post. If you think I should summarize the whole .NET piece of this in our blog post **instead** of linking people over here, I'm okay with that too, just let me know your opinion.